circle.yml

```diff
@@ -4,19 +4,8 @@ machine:
 
 test:
   override:
-    - npm run test:ci
+    - npm run danger
 
 general:
   artifacts:
     - "coverage/lcov-report"
-
-dependencies:
```

Comment:
CircleCI does not allow you to selectivity expose sensitive environment variables to certain branches only. So for security reasons, I had to disable auto deployment to `npm`.

Reply:
https://discuss.circleci.com/t/env-variables-fro-different-branches/3672/4

Reply:
We could patch this with a little `now` deployment of some [webhook responder](https://github.com/nlf/node-github-hook), if it would really help your preferred workflow.

Reply:
@zetlen that would have a similar problem. CI would need some private token to communicate to our custom service that something is ready for deployment. If someone has access to run code in CI, they could invoke that themselves with a malicious changeset.

Reply:
Yeah, we could do that at some point. Other option would be to migrate to Travis CI, which has support for this.